### DISCUSSION TITLE
filters filtered by what is in the library

### DISCUSSION THREAD

**Discussion by jhurlbut** [okio-ai/nendo-platform#21] [Ideas]
It would be helpful if the filter option only provided checkboxes for values that are existing in the library. For example if I filter by sfx I should only see those sfx categories that actually exist in tracks in the library and not a bunch of random sfx that have no associated track

**Comment by faradox** [comment]
Yes, we already have an item on our feature wishlist for that. But the implementation of it is not trivial. Will notify you in this thread once we have a solution in place.
